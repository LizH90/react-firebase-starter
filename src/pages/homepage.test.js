import React from "react"
import renderer from "react-test-renderer"
import HomePage from "./homepage"

test("The homepage renders correctly", () => {
  const component = renderer.create(<HomePage />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
