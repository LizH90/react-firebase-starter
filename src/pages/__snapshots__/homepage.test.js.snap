// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The homepage renders correctly 1`] = `
Array [
  <h1>
    Welcome 
  </h1>,
  <div
    className="App"
  >
    <header
      className="App-header"
    >
      <button
        onClick={[Function]}
        type="submit"
      >
        Sign Out
      </button>
    </header>
  </div>,
]
`;
